test/test0028/assignment.c

```diff
@@ -15,10 +15,14 @@
  * along with M2-Planet.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+const int unused;
+
 int main() {
+    const int one = 1;
+    const int three = 3;
```

Comment:
identation. I think the rest of the file uses tabs.

Reply:
Fixed.